meshtastic/module_config.proto

```diff
@@ -583,6 +583,26 @@ message ModuleConfig {
      */
     NeighborInfoConfig neighbor_info = 10;
   }
+
+  /*Ambient Lighting Module - Settings for control of onboard LEDs to allow users to adjust the brightness levels and respective color levels.
+Initially created for the RAK14001 RGB LED module.
+*/
+
+message AmblientLightingConfig {
+
+/*Sets the overall current for the LED, range for the RAK14001 is 1-31*/
```

Comment:
Looks like the indents for this message and comments are off

Reply:
Fixed